specification/security/resource-manager/Microsoft.Security/preview/2024-07-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1707 @@
+        },
+        "scanInterval": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Scan interval in hours (value should be between 1-hour to 24-hours)"
+        }
+      }
+    },
+    "AwsOrganizationalData": {
+      "type": "object",
+      "description": "The AWS organization data",
+      "discriminator": "organizationMembershipType",
+      "required": [
+        "organizationMembershipType"
+      ],
+      "properties": {
+        "organizationMembershipType": {
+          "type": "string",
+          "description": "The multi cloud account's membership type in the organization",
+          "enum": [
+            "Member",
+            "Organization"
+          ],
+          "x-ms-enum": {
+            "name": "organizationMembershipType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "AwsOrganizationalDataMaster": {
+      "type": "object",
+      "description": "The AWS organization data for the master account",
+      "x-ms-discriminator-value": "Organization",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AwsOrganizationalData"
+        }
+      ],
+      "properties": {
+        "stacksetName": {
+          "type": "string",
+          "description": "If the multi cloud account is of membership type organization, this will be the name of the onboarding stackset"
+        },
+        "excludedAccountIds": {
+          "type": "array",
+          "description": "If the multi cloud account is of membership type organization, list of accounts excluded from offering",
+          "items": {
+            "type": "string",
+            "description": "account IDs"
+          }
+        }
+      }
+    },
+    "AwsOrganizationalDataMember": {
+      "type": "object",
+      "description": "The AWS organization data for the member account",
+      "x-ms-discriminator-value": "Member",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AwsOrganizationalData"
+        }
+      ],
+      "properties": {
+        "parentHierarchyId": {
+          "type": "string",
+          "description": "If the multi cloud account is not of membership type organization, this will be the ID of the account's parent"
+        }
+      }
+    },
+    "GcpProjectEnvironmentData": {
+      "type": "object",
+      "description": "The GCP project connector environment data",
+      "x-ms-discriminator-value": "GcpProject",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EnvironmentData"
+        }
+      ],
+      "properties": {
+        "organizationalData": {
+          "type": "object",
+          "description": "The Gcp project's organizational data",
+          "$ref": "#/definitions/GcpOrganizationalData"
+        },
+        "projectDetails": {
+          "type": "object",
+          "description": "The Gcp project's details",
+          "$ref": "#/definitions/GcpProjectDetails"
+        },
+        "scanInterval": {
```

Comment:
scanIntervalInHours?

Reply:
This version is build on top of the older version with the goal of supporting a new environment only, without the intention to make further unrelated changes

Reply:
This version is build on top of the older version with the goal of supporting a new environment only, without the intention to make further unrelated changes.
And an private PR already reviewed and approved by your team, please see it here https://github.com/Azure/azure-rest-api-specs-pr/pull/18430